src/test/kotlin/org/rust/lang/refactoring/IntroduceVariableTest.kt

```diff
@@ -0,0 +1,60 @@
+package org.rust.lang.refactoring
+
+import org.rust.lang.RustTestCaseBase
+import org.rust.lang.core.psi.RustExprStmtElement
+
+class IntroduceVariableTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/lang/refactoring/fixtures/introduce_variable/"
+
+//    @Ignore
+//    fun testVariable() = checkByFile {
+//        val rustLocalVariableHandler = RustLocalVariableHandler()
+//        openFileInEditor("variable.rs")
+//        rustLocalVariableHandler.invoke(myFixture.project, myFixture.editor, myFixture.file, com.intellij.openapi.actionSystem.DataContext.EMPTY_CONTEXT)
+//    }
+
+
+    fun testMultipleOccurrences() = checkByFile {
```

Comment:
Let's try avoid fixture files here. Take a look at these tests: https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/ide/typing/RustEnterInStringLiteralHandlerTest.kt#L8

Here, we use multiline string literal to embed Rust code inside Kotlin, which is a bit more convenient.